Client/Frontend/Settings/Rewards Internals/RewardsInternalsViewController.swift

```diff
@@ -132,8 +131,7 @@ class RewardsInternalsViewController: TableViewController {
               text: Strings.RewardsInternals.walletPaymentID, detailText: internals.paymentId,
               selection: { [unowned self] in
                 if let index = self.dataSource.sections[safe: legacyWalletSection]?.rows.firstIndex(where: { $0.cellClass == PaymentIDCell.self }),
-                  let cell = self.tableView.cellForRow(at: IndexPath(item: index, section: legacyWalletSection)) as? PaymentIDCell
-                {
+                  let cell = self.tableView.cellForRow(at: IndexPath(item: index, section: legacyWalletSection)) as? PaymentIDCell {
```

Comment:
one space off

Reply:
Unfortunately they're all gonna be like this because of the original formatter when I did 2 space… should I try reformatting to fix that?

Reply:
oh ok dont bother 😛